python/cudf/cudf/core/series.py

```diff
@@ -155,10 +155,14 @@ def values(self):
         if len(self) == 0:
             return cupy.asarray([], dtype=self.dtype)
 
-        return cupy.asarray(
-            # Temporary fix for CuPy < 7.0, numba = 0.46
-            numbautils.PatchedNumbaDeviceArray(self.to_gpu_array())
+        vals = (
+            self.fillna(False).data.mem
+            if self.null_count != 0
+            else self.data.mem
```

Comment:
So this is in the `.values` property which returns a device array from a Series. This will make it so that if there's nulls in the Series it returns `False` values in the array which doesn't make sense because you could likely already have `False` values in the Series. We need to just assert there's no nulls here instead.